portal-kernel/src/com/liferay/portal/kernel/servlet/MultiSessionErrors.java

```diff
@@ -63,6 +64,25 @@ public static boolean isEmpty(PortletRequest portletRequest) {
 		return false;
 	}
 
+	public static boolean isHideDefaultErrorMessage(
+		PortletRequest portletRequest, String portletId) {
+
+		String key =
+			portletId + SessionMessages.KEY_SUFFIX_HIDE_DEFAULT_ERROR_MESSAGE;
```

Comment:
Minor SF: There is no need for `key` var, this can be inline on L73.

Reply:
Got it, went ahead and inlined it.